### PR TITLE
Null fields

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
Fields with a null value in the response must be parsed correctly. I have added `allow_none=True` to the fields that the documentation says are optional or can be null. Some models have been updated with additional fields that were missing.

### CODE REVIEW

File: connect/models/event.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```
Comment (vgrebenschikov) [inline connect/models/event.py]: Commit message "Null fields" does not really matches support of events.

Comment (JaviCerveraIngram) [reply connect/models/event.py]: They have simply been moved from other source files into their own file.

File: connect/models/event.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from connect.models.company import CompanySchema, Company
+
+
+class EventInfo(BaseModel):
+    at = None  # type: Optional[str]
+    by = None  # type: Optional[Company]
+
+
+class EventInfoSchema(BaseSchema):
+    at = fields.Str(allow_none=True)
```
Comment (ht-albert) [inline connect/models/event.py]: I think it's datetime type

File: connect/models/product.py [hunk 3]
```diff
@@ -6,36 +6,127 @@
...
 
+class Renewal(BaseModel):
+    from_ = None  # type: str
+    to = None  # type: str
+    period_delta = None  # type: int
+    period_uom = None  # type: str
+
+
+class RenewalSchema(BaseSchema):
+    from_ = fields.Str(attribute='from')
```
Comment (ht-albert) [inline connect/models/product.py]: It is too datetime type

File: connect/models/event.py (hunk 2 above)
Comment (JaviCerveraIngram) [reply connect/models/event.py]: Thanks. Will fix it tomorrow.

**Pull request merged**